docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '3'
+services:
+    
+    dalalserver:
+        container_name: dalalstreet-server
+        build: .
+        volumes: 
+            - ./dalalstreet_docker.log:/dalal-street-server/dalalstreet_docker.log
```

Comment:
Check the logs directory once. Refer (this)[https://github.com/delta/dalal-street-server/blob/d7b5145a593afb997f7599e73656843aac1cca54/docker-compose.yml#L18]